packages/uxp-devtools-cli/src/cli/utils/CLICDTInspectMgr.js

```diff
@@ -18,6 +18,7 @@ const { createDeferredPromise } = require("./common");
 
 function getDevtoolsAppExecutablePath() {
     const productName = "Adobe UXP Developer Tool";
+    console.log('hehe')
```

Comment:
He?

Reply:
Didn't realize I still had this PR open 😄